components/taupage.rst

```diff
@@ -400,6 +400,28 @@ IAM-Role::
        ]
      }
 
+.. _iamEraseOnBootTag:
+
+IAM-Role for use with **Taupage:erase-on-boot** tag on the EBS volume::
```

Comment:
@Raffo that's valid RST format, but it should probably changed to `..code-block:: yaml`